Simplenote/src/main/java/com/automattic/simplenote/utils/NoteUtils.java

```diff
@@ -75,8 +75,13 @@ public void onClick(DialogInterface dialog, int whichButton) {
 
                             // Show empty placeholder for large devices in landscape.
                             if (activity instanceof NotesActivity) {
-                                ((NotesActivity) activity).showDetailPlaceholder();
-                            // Close editor for small devices and large devices in portrait.
+                                NotesActivity notesActivity = (NotesActivity) activity;
+                                if (notesActivity.getNoteListFragment() != null) {
+                                    notesActivity.getNoteListFragment().updateSelectionAfterTrashAction();
+                                } else {
+                                    notesActivity.showDetailPlaceholder();
+                                }
+                                // Close editor for small devices and large devices in portrait.
```

Comment:
```suggestion
                            // Close editor for small devices and large devices in portrait.
```

This comment was indented with the checkstyle automation.  Let's move it back one indention so that it is the way it was before this pull request and it is clearer the comment is referring to the `else if` below.

Reply:
Done 6b925da8d3374e1775ea5f803589f51670aa9860, but IMO those comments should go after the curly brace of the `else if` not before, and this would've avoided this indentation issue 🙂